package.json

```diff
@@ -129,6 +129,7 @@
   ],
   "dependencies": {
     "@exchangeunion/grpc-dynamic-gateway": "^0.3.10",
+    "bignumber.js": "^9.0.0",
```

Comment:
I'm not sure about adding this dependency. Is this really necessary?

Reply:
I didn't find more elegant and correct way without checking out corner cases to check if Number decimal places are at most 12.

Reply:
I agree that this is not necessary, see my comment above about a simpler way to check if we have more than 12 decimal places.